lib/production_e2e_filter.rb

```diff
@@ -0,0 +1,43 @@
+module ProductionE2EFilter
+
+  BEGIN {
+    logger = Logger.new(STDOUT)
```

Comment:
don't know such logging utility - can you have a log tag to this logs?
do you want maybe to add some configuration to enable/disable this in prod?
are you sure you do not want to inject the domains/emails as env var?

Reply:
I added a configuration to enable/disable in prod (as an env variable)

In terms of the domains/emails I'm not sure how to inject them as an env var because if I understand correctly env vars can't be arrays, only key value pairs. Let me know if I'm wrong about that

for the logger, this seems to be what the yLogger uses under the hood, while adding a log_tag field. I implemented a similar mechanism here, let me know what you think

Reply:
you can inject a comma separated sting, and parse it into an array using a simple string.split(',') function.
suppling this YAML file as part of the hosting GIT project sounds to me like an exhausting operation. 
lets change it into env var